apps/tsilhqotin-language-hub/src/Components/Pages/Song/Song.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card } from '@mui/material';
+
+export type SongData = {
+    id: string;
+
+    title: string;
+
+    titleEnglish?: string;
+
+    contributions: string;
+
+    lyrics: string;
+
+    audioURL: string;
+};
+
+/* eslint-disable-next-line */
+export interface SongsDetailComponentProps {
+    songData: SongData;
+}
+
+export function Song(props: SongsDetailComponentProps) {
+    const { songData } = props;
+
+    const { id, title, titleEnglish, contributions, lyrics, audioURL } = songData;
+
+    return (
+        <div>
+            <div id="heading">
+                <div id="container">
+                    <h1 id="pageTitle"> {`${title || titleEnglish}`}</h1>
+                </div>
+            </div>
+            <Card>
+                <div>English: {titleEnglish}</div>
```

Comment:
One minor issue here. `title` and `titleEnglish` are both optional. Do you recall the presentational pipe we had in Angular for this situation? I'd create a helper function `createFullTitle = (title: string, titleEnglish: string): string =>{
// logic here
} and then call that instead of `${title || titleEnglish}`.

The logic would check the 3 possibilities:
Title In Language (English Title)
Title In Language
English Title
and throw if Title and Title English are undefined (Should never happen because of the backend invariant validation logic). Eventually, this is the kind of error you'd want to catch in an error boundary- but just trust the backend for now.